hugr/src/ops/leaf.rs

```diff
@@ -13,8 +13,14 @@ use crate::{
     types::{EdgeKind, FunctionType, Type, TypeRow},
 };
 
-/// A user-defined operation that can be downcasted by the extensions that
-/// define it.
+/// A user-defined operation defined in an extension.
+///
+/// Any custom operation can be encoded as a serializable [`OpaqueOp`].
+/// If the operation's extension is loaded in the current context, the operation
+/// can be resolved to an executable [`ExtensionOp`].
```

Comment:
"executable" doesn't seem right, don't we just mean that the extensionop refers to it's definition in the extension?

Reply:
That sounds better :+1: